pkg/verifier/duplicated_compaction.go

```diff
@@ -28,7 +28,7 @@ func (DuplicatedCompactionBlocks) VerifyRepair(ctx Context, idMatcher func(ulid.
 		return errors.Errorf("id matching is not supported")
 	}
 
-	level.Info(ctx.Logger).Log("msg", "started verifying issue", "with-repair", repair)
+	_ = level.Info(ctx.Logger).Log("msg", "started verifying issue", "with-repair", repair)
```

Comment:
I think a better option would be to do this :
```go
err = level.Info(ctx.Logger).Log("msg", "started verifying issue", "with-repair", repair)
if err!=nil {
   return err
}
```
Ignoring the error may cause issues.

Reply:
Will it be correct to return the error?

Reply:
yes, similar to this https://github.com/thanos-io/thanos/blob/main/pkg/verifier/duplicated_compaction.go#L34

Reply:
Okay. Will update it. Thank you!